charts/victoria-metrics-k8s-stack/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Next release
 
 - Moved dashboards templating logic out of sync script to Helm template
+- Support importing jsonnet dashboards and rules
```

Comment:
I would suggest to mention the change which is useful for users:
```suggestion
- Synchronize Etcd dashboards and rules with mixin provided by Etcd.
```

The fact that it is now possible to import `.jsonnet` is most likely not very useful to the end users, but only to the chart developers.